app/load_container_filters.go

```diff
@@ -0,0 +1,66 @@
+package app
```

Comment:
I'm not sure it's worth having a separate module just for this, particularly once you simplify the id generation.